networks/rpc/utils.go

```diff
@@ -247,3 +247,38 @@ func sanitizeTimeouts(timeouts HTTPTimeouts) HTTPTimeouts {
 
 	return timeouts
 }
+
+func SchemeToString(scheme ServerScheme) string {
```

Comment:
How about managing this with map[ServerScheme]string?

Reply:
Is there any good stuff by creating a global map instead of a function call?

Reply:
I guess performance? (function call overhead vs. map lookup) But I'm not sure if it's significant, so if you don't agree, you can disregard this comment.

Reply:
Okay. the performance seems not to matter in my view. Function seems more flexible to extend (e.g., log invalid input if the parameter is not an expected type or value). Let me keep it as function.